src/calculateRentalCost.js

```diff
@@ -5,6 +5,24 @@
  */
 function calculateRentalCost(days) {
   // write code here
+
+  const perDay = 40;
+  const longDay = 7;
+  const shortDay = 3;
+  const discLongDay = 50;
+  const discShortDay = 20;
+
+  const total = days * perDay;
+
+  if (days >= longDay) {
+    return total - discLongDay;
+  }
+
+  if (days >= shortDay && days < longDay) {
+    return total - discShortDay;
```

Comment:
The check `days < longDay` is unnecessary because if `days` is greater than or equal to `longDay`, the function would have already returned in line 18. You can remove this condition to simplify the code.

Reply:
AI is right here too, just remove the second check in your _if_ statement (from && onwards)